common/src/test/java/com/revenuecat/purchases/common/AnonymizerTest.kt

```diff
@@ -0,0 +1,75 @@
+package com.revenuecat.purchases.common
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.annotation.Config
+
+@RunWith(AndroidJUnit4::class)
+@Config(manifest = Config.NONE)
+class AnonymizerTest {
+    private lateinit var anonymizer: Anonymizer
+
+    @Before
+    fun setup() {
+        anonymizer = Anonymizer()
+    }
+
+    // region anonymizeString
+
+    @Test
+    fun `anonymizeString removes emails`() {
+        val originalString = "Some random text with an sample.123+34@revenuecat.com and test.1@gmail.com email."
```

Comment:
👍🏻 I was gonna suggest having a relatively complex e-mail to cover the fact that you used a good regex.
It's probably not perfect, but this is probably covering 99% of e-mails.